README.md

```diff
@@ -16,3 +16,79 @@ Create a `.env` file. See `.env.example` for environment variables which may be
+| GeneralizedTCR | [0xb72103eE8819F2480c25d306eEAb7c3382fBA612](https://etherscan.io/address/0xb72103eE8819F2480c25d306eEAb7c3382fBA612#code) |
+| CPKFactory | [0x0fB4340432e56c014fa96286de17222822a9281b](https://etherscan.io/address/0x0fB4340432e56c014fa96286de17222822a9281b#code) |
+| Uniswap V2 Factory | [0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f](https://etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#code) |
+| WETH9 | [0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2#code) |
+| GelatoCore | [0x025030BdAa159f281cAe63873E68313a703725A5](https://etherscan.io/address/0x025030BdAa159f281cAe63873E68313a703725A5#code) |
+
+
+## Rinkeby Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0x36bede640D19981A82090519bC1626249984c908](https://rinkeby.etherscan.io/address/0x36bede640D19981A82090519bC1626249984c908#code)     |
+| [FPMMDeterministicFactory](https://github.com/gnosis/conditional-tokens-market-makers) | [0x0fB4340432e56c014fa96286de17222822a9281b](https://rinkeby.etherscan.io/address/0x0fB4340432e56c014fa96286de17222822a9281b#code) |
+| [Reality.eth](https://github.com/realitio/realitio-contracts) | [0x3D00D77ee771405628a4bA4913175EcC095538da](https://rinkeby.etherscan.io/address/0x3D00D77ee771405628a4bA4913175EcC095538da#code) |
+| Reality.eth Oracle Adapters | [0x17174dC1b62add32a1DE477A357e75b0dcDEed6E](https://rinkeby.etherscan.io/address/0x17174dc1b62add32a1de477a357e75b0dcdeed6e#code) |
+| Reality.eth Oracle Adapters (Scalar) | [0x0e8Db8caD541C0Bf5b611636e81fEc0828bc7902](https://rinkeby.etherscan.io/address/0x0e8Db8caD541C0Bf5b611636e81fEc0828bc7902#code) |
+| DXTokenRegistry | [0x03165df66d9448e45c2f5137486af3e7e752a352](https://rinkeby.etherscan.io/address/0x03165df66d9448e45c2f5137486af3e7e752a352#code) |
+| GeneralizedTCR | [0x3b29096b7ab49428923d902cEC3dFEaa49993234](https://rinkeby.etherscan.io/address/0x3b29096b7ab49428923d902cec3dfeaa49993234#code) |
+| CPKFactory | [0x336c19296d3989e9e0c2561ef21c964068657c38](https://rinkeby.etherscan.io/address/0x336c19296d3989e9e0c2561ef21c964068657c38#code) |
+| Uniswap V2 Factory | [0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f](https://rinkeby.etherscan.io/address/0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f#code) |
+| WETH9 | [0xc778417E063141139Fce010982780140Aa0cD5Ab](https://rinkeby.etherscan.io/address/0xc778417E063141139Fce010982780140Aa0cD5Ab#code) |
+| GelatoCore | [0x733aDEf4f8346FD96107d8d6605eA9ab5645d632](https://rinkeby.etherscan.io/address/0x733aDEf4f8346FD96107d8d6605eA9ab5645d632#code) |
+
+
+## xDai Network
+
+| Contract | Address |
+| -------- | -------- |
+| [ConditionalTokens](https://github.com/gnosis/conditional-tokens-contracts) | [0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce](https://blockscout.com/poa/xdai/address/0xCeAfDD6bc0bEF976fdCd1112955828E00543c0Ce/read-contract)     |
```

Comment:
For the sake of consistency i would add here so that it point to /contracts like rest